### ISSUE TITLE
Strange artifact near Spica

### ISSUE THREAD

**Issue by Gnarflord** [Stellarium/stellarium-web-engine#159]
So I was recently playing around in Stellarium Web and found something interesting:

![Screenshot from 2022-07-31 20-50-05](https://user-images.githubusercontent.com/19466442/182041086-49f65a9e-5fa9-4c74-b4ab-662b1da3e323.png)
![Screenshot from 2022-07-31 20-50-11](https://user-images.githubusercontent.com/19466442/182041090-1a477110-aeb7-4c48-b6f1-c9ab2995405f.png)
![Screenshot from 2022-07-31 20-50-25](https://user-images.githubusercontent.com/19466442/182041091-5f99f9a3-62f5-4e63-b208-87608229f9ca.png)

If this turns out to be the first confirmed UFO sighting I'd like to call dibs ;)

For realsies: I suspect this to be an artifact from the original telescope data used to create the background. Maybe some cursed optics caused the lense to reflect parts of the internal telescope structure? I've got no clue. This is also visible in the mobile app, but not in the standalone OpenGL Stellarium application.

Time of recording: 2022-07-31 22:45:45 UTC
Coordinates: 49°50′N 9°50′E (rounded, Internet people are creeps sometimes)

**Comment by gzotti** [comment]
Yes, the well-known instrument-internal reflection visible near every bright star, here caused by nearby Spica.

**Comment by Gnarflord** [comment]
Oh sorry for the obvious question then, now I feel stupid.
But out of curiosity: I've only recently started using the App instead of the standalone application, why is this only visible in the Stellarium Web Engine?

Also in some areas there seems to be even additional bright straight lines, like here next to Arcturus:

![image](https://user-images.githubusercontent.com/19466442/182045869-a48f24e2-8a43-4c4f-95dc-3faf3bd2368c.png)

**Comment by gzotti** [comment]
Stellarium (desktop) and the Stellarium Web Engine are entirely different programs. Stellarium (desktop) simulates stars from a catalog, usually without all-sky photo coverage. You can however activate the DSS survey or HiPS surveys (see User Guide for details). SWE automatically shows a photographic background when you zoom in enough. 

The lines may be meteors or satellites or probably just further reflections, like this blue caustic monstrosity within the mirror pattern. The satellite craze which is now going on means we will probably no longer be able to capture wide-field astrophotos without satellite streaks.

**Comment by MtzRJ** [comment]
Near to scorpion

Em dom., 31 de jul. de 2022 18:26, Gnarflord ***@***.***>
escreveu:
